src/Constants.cpp

```diff
@@ -47,16 +47,14 @@ const double MAX_DTHETA = kinematics::DiffDriveKinematics(EFF_WHEEL_BASE)
 							  .wheelVelToRobotVel(-MAX_WHEEL_VEL, MAX_WHEEL_VEL)(2);
 
 // TODO: We need to recalibrate the camera, since we replaced it with a different one.
-// TODO: rename cameras (in MC as well) as appropriate
 const char* MAST_CAMERA_CONFIG_PATH = "../camera-config/MastCameraCalibration.yml";
-const robot::types::CameraID MAST_CAMERA_ID =
-	"upperArm"; // TODO: replace with real camera name
+const robot::types::CameraID MAST_CAMERA_ID = "mast"; // was upperArm
```

Comment:
No need to put the `// was upperArm` comment on this and the other definitions I think. We can document that on this PR if necessary